geocoder_module/geocoder.py

```diff
@@ -280,7 +274,9 @@ def get_location_info(
                     continue
         if validated_results == []:
             validated_results = [{}]
-            logging.warning(f"Location validation failed for {location}")
+            logging.warning(
```

Comment:
Shall we use the same syntax throughout the whole codebase? eg: 
```suggestion
            logging.warn(
```

Reply:
Acutally ignore the above please - apparently we should be using `warning` instead, so please use that! As `warn` is depreciated.